src/hooks/useFormData.ts

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+
+type Address = {
+  prefecture: string;
+  area: string;
+};
+
+export type AddressSearchResult = {
+  results: {
+    address1: string;
+    address2: string;
+  }[];
+};
+
+export const useFormData = () => {
+  const [address, setAddress] = useState<Address>({ prefecture: '', area: '' });
```

Comment:
Is this state necessary? I noticed you haven't used it.

Reply:
Removed unused address state